### DISCUSSION TITLE
[Solved] Custom Links for Privacy Policy and Imprint

### DISCUSSION THREAD

**Discussion by l4rm4nd** [docusealco/docuseal#272] [Ideas]
Selhosters running DocuSeal may want to provide links to a privacy policy, cookie policy and a general imprint.

Currently, none of such information is included when selfhosting DocuSeal. This may lead to legal problems if the software is used by third parties that are not made aware about those topics. I am living in the EU, where such things are mandatory to outline.

I propose a new feature that allows defining custom URLs to such documents. The Privacy Policy, Cookie Policy and Imprint links are then displayed in the footer of DocuSeal, if defined by the selfhoster.

Typically, I would recommend implementing those by setting environment variables.

**Comment by omohokcoj** [comment]
@l4rm4nd custom links and text can be added to the signing form in the field description.
For example you can add a checkbox field and put your privacy/imprint links in the field description.
Let me know if that's sufficient for your use case.

![image](https://github.com/docusealco/docuseal/assets/5418788/fbc274a1-3208-4c41-9c3e-5e67e258cb3a)
![image](https://github.com/docusealco/docuseal/assets/5418788/bc02b690-8ec5-4abd-a0df-71bdd32cba7a)

**Reply by l4rm4nd** [reply]
Hi @omohokcoj ,

many thanks for the feedback. That's a first good approach for adding privacy links and checkboxes into a fillable form itself.

My proposal would rather aim at the docuseal web application itself though. So if someone browses the root page, not via an unique form link from email or so, there should be a general privacy policy too.

So basically just two hyperlink placeholders someone can set. The root page and a signing page itself too would then display the links in the footer.

If someone questions how the docuseal website stores and handles data, the links would provide the necessary information. As on any other website via footer links to privacy policy, cookies etc.

To keep it simple, it would just be a hyperlink. No feature to put in real text, HTML or so. So selfhosters can point to another site (e.g. company homepage or so), which hosts the privacy policy etc.

I would like to keep the signable forms itself clean from such legal, required links etc.

**Comment by l4rm4nd** [comment]
Just referencing this, as it summarizes perfectly what I need and meant by this discussion:

https://github.com/docusealco/docuseal/issues/306
